app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="header_playlists">Afspeellijsten</string>
+    <string name="header_songs">Nummers</string>
```

Comment:
Is this the right translation? Google Translate is saying this means "Numbers"

Reply:
Well, the literal translation of "Nummers" is indeed "Numbers", but we also use it for "Songs".

"Songs" gets translated to either "Nummers" or "Liedjes"